extensions/wikia/Insights/models/InsightsUnconvertedInfoboxesModel.php

```diff
@@ -43,9 +43,16 @@ public function hasAltAction() {
 	}
 
 	public function getAltActionUrl( Title $title ) {
-		$subpage = Title::newFromText( $title->getText() . "/Draft", NS_TEMPLATE );
+		if ( !class_exists( 'TemplateConverter' ) ) {
+			return '';
```

Comment:
Returning an empty string will still cause the button to appear with an empty `href` attribute. What about adding a check [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/Insights/models/InsightsQuerypageModel.php#L410)?